exl_snap_backup

```diff
@@ -18,18 +18,23 @@ APP_VGNAME=vg_voyager
 APP_LVNAME=`lvs | grep ${APP_VGNAME} | awk '{ print $1 }' | egrep -v "backup|snap"`
 
 TODAY=`date +%Y%m%d`
+DAYSAGO=`date -d 'now - 1 days' "+%Y%m%d"`
```

Comment:
@sgurnick This is a YYYYMMDD string, but the name `DAYSAGO` makes me expect a number like 2 or 6.